src/test/java/it/pagopa/transactions/utils/UUIDUtilsTest.java

```diff
@@ -27,4 +28,12 @@ void shouldEncodeAndDecodeBase64OfUUID() {
 
         assertEquals(uuid, uuidFromBase64.get());
     }
+
+    @Test
+    void shouldDecodeBase64OfUUIDError() {
+        String wrongUuid = "xxxx";
+        Either<InvalidRequestException, UUID> uuidFromBase64 = uuidUtils.uuidFromBase64(wrongUuid);
+        assertEquals(InvalidRequestException.class, uuidFromBase64.getLeft().getClass());
```

Comment:
This line is not actually needed as it is guaranteed from the type signature, a `assertTrue(uuidFromBase64.isLeft())` should suffice

Reply:
you are right !!!! 👀